src/gatk_gcnv_caller.sh

```
@@ -437,9 +437,13 @@ _call_generate_gcnv_bed() {
     denoised_copy_ratio_files=$(find inputs/vcfs/ -name "*_denoised_copy_ratios.tsv")
 
     SECONDS=0
+
+    keep_samples_arg=""
+    if [[ "$keep_all_sample_traces" == "true" ]]; then keep_samples_arg="--keep_all_samples "; fi
```

Shellcheck

[warning] keep_all_sample_traces is referenced but not assigned.

(SC2154)